baybe/utils/dataframe.py

```diff
@@ -184,6 +189,13 @@ def add_fake_results(
 
     # Add the fake data for each target
     for target in targets:
+        if isinstance(target, BinaryTarget):
+            # TODO: When refactoring, take into account good and bad intervals
```

Comment:
is this a complex thing or why have you decided to already imply an upcoming refactoring?

Reply:
No, not a complex thing, but I'm already starting to run out of time before my vacation and want to avoid last minute coding the night before I hit the road 😄 If you prefer to have it in this PR, feel free to add the functionality 👍🏼 

Regarding "implied" refactoring: just referring to the one obvious outstanding refactoring of the simulation module, where there are still so many half-broken things (like the callable lookup etc). To me, this all goes under "one big mess" 🙈

Reply:
Also, how does the concept of having "good and bad intervals" translate to binary targets?

Reply:
the intervals are with respect to the inputs. so that's ok, but still needs to be implemented. I'll skip it for this PR though, no time left